components/global_header/left_controls/product_menu/product_menu_list/index.ts

```diff
@@ -75,8 +78,8 @@ function mapStateToProps(state: GlobalState) {
         firstAdminVisitMarketplaceStatus: getFirstAdminVisitMarketplaceStatus(state),
         showVisitSystemConsoleTour,
         enableCustomUserGroups,
-        isCloud,
-        isFreeTrial,
+        isStarterFree,
+        isStarterFreeTrial,
```

Comment:
Can we keep the `isFreeTrial` variable name? I think its clearer.

Reply:
done